lib/reline/line_editor.rb

```diff
@@ -880,6 +880,7 @@ def editing_mode
         @perfect_matched = completed
       else
         @completion_state = CompletionState::MENU
+        complete(list, just_show_list) if @config.show_all_if_ambiguous
```

Comment:
I think the flow is complex because complete became a recursive method.
How about `complete_internal_proc(list, true)`

Reply:
Since `complete_internal_proc` does not change the state of `@completion_state`, it did not fit this purpose. I decided to call it `complete(list, true)`.